core/shared/src/main/scala/interface.scala

```diff
@@ -123,9 +116,45 @@ trait Param[Typeclass[_], Type] extends Serializable {
     *  [[scala.collection.Seq]] to hide the mutable collection API. */
   final def annotations: Seq[Any] = annotationsArray
 
+  override def toString: String = s"ReadOnlyParam($label)"
+}
+
+/** represents a parameter of a case class
+  *
+  *  @tparam Typeclass  type constructor for the typeclass being derived
+  *  @tparam Type       generic type of this parameter */
+trait Param[Typeclass[_], Type] extends ReadOnlyParam[Typeclass, Type] {
+
+  /** provides the default value for this parameter, as defined in the case class constructor */
+  def default: Option[PType]
+
   override def toString: String = s"Param($label)"
 }
 
+// FIXME: docs
```

Comment:
I'd say either fix it or remove it 😄

Reply:
Removal is fine by me. I'm very much of the opinion that documentation shouldn't hold up code.

Reply:
My own convention is that fixme's are to be dealt with in the PR so I'll fix this up :)